src/test/java/README.md

```diff
@@ -0,0 +1 @@
+test files go here
```

Comment:
I would add a base test class instead as a stub here, where we can start adding common methods as they are needed. So the structure here will be adding a `core/src/test/java/org/htsjdk/core` as following:

```java
/**
 * Base test class for HTSJDK.
 * 
 * <p>All tests should implement this base class.
 */
public class HtsjdkBaseTest {

}
````